docs/observability/reliability-management-slo/dashboards.md

```diff
@@ -62,12 +62,23 @@ Each dashboard contains the following information:<br/><img src={useBaseUrl('img
 </table>
 
 
-## Setting Granularity for SLO Time Ranges
+## Setting SLO Data Granularity
+
+### Time Ranges
 
 To modify the time range, select and drag across dates to zoom in further. This can be useful if you want to zoom in for granular details, especially for charts with larger compliance periods:<br/><img src={useBaseUrl('img/observability/2022-08-02_14-29-19.gif')} alt="Reliability Management SLO SLI" />
 
+### Compliance Period
+
 You can also filter by compliance period to view your past activity and plan ahead:<br/><img src={useBaseUrl('img/observability/compliance-period-filter.png')} alt="Reliability Management SLO SLI" />
```

Comment:
```suggestion
You can also filter by compliance period to view your past activity and plan ahead.<br/><img src={useBaseUrl('img/observability/compliance-period-filter.png')} alt="Reliability Management SLO SLI" />
```

Reply:
@kimsauce: Is there any specific reason for using ":" here? I see it in the above sentence as well.

Reply:
Colon precedes an image, like this:
<img width="635" alt="Screenshot 2023-02-07 at 12 20 22 PM" src="https://user-images.githubusercontent.com/56411016/217356263-a92a21a7-0a75-40d3-aea2-a854b07c75b4.png">

Reply:
That's a complete sentence and we can't put colon here. It's rule in grammar and the reason is colon and em dashes are always used to signal further clarification or used with lists.

Reply:
Per the example in my previous comment, this sentence (and the ones like it throughout the docs) are introducing images and code snippets. This formatting is used everywhere in tech docs (Google, Microsoft, GitHub, etc.). Can you please add to Docs Meeting agenda for discussion?